src/components/InputField.tsx

```diff
@@ -18,6 +18,8 @@ type Props = {
   defaultValue?: string
   value?: string
   numberOfLines?: number
+  bottom?: boolean
```

Comment:
At a glance what `top` and `bottom` means is unclear to me. Maybe we could call them `topRadius` and `bottomRadius`? Another pattern I've been using for rounding corners is `isFirst` and `isLast`.

Reply:
Good point, I like isFirst, isLast. Let me update